bee-api/bee-rest-api/src/types/responses.rs

```diff
@@ -54,8 +74,22 @@ pub struct ProtocolResponse {
     pub min_pow_score: f64,
     #[serde(rename = "rentStructure")]
     pub rent_structure: RentStructureResponse,
-    #[serde(rename = "tokenSupply")]
-    pub token_supply: String,
+    #[serde(rename = "baseToken")]
+    pub base_token: BaseTokenResponse,
+}
+
+/// Returned in [`InfoResponse`].
+/// Rent information about the node.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct BaseTokenResponse {
+    pub name: String,
+    #[serde(rename = "tickerSymbol")]
+    pub ticker_symbol: String,
+    pub unit: String,
+    pub decimals: u8,
+    pub subunit: String,
```

Comment:
Maybe change the order?
```suggestion
    pub unit: String,
    pub subunit: String,
    pub decimals: u8,
```